docs/content/tutorials/kic/deploy-with-kic.md

```diff
@@ -399,40 +397,17 @@ To enable metrics collection for the NGINX OSS Ingress Controller, take the foll
       nsm.nginx.com/daemonset: <name of NGINX Ingress Controller DaemonSet>
       ```
 
-   This allows metrics scraped from NGINX OSS Ingress Controller Pods to be associated with the resource that created the Pods.
+   This allows metrics scraped from NGINX Ingress Controller Pods to be associated with the resource that created the Pods.
 
-### NGINX Plus Ingress Controller Metrics
+#### Enabling NGINX Plus Ingress Controller Metrics
```

Comment:
I think we can just remove this entire section. In the section above, simply add a note following the first step that says something like "If using N+ controller, add this additional flag (enable-latency-metrics). Everything else between these sections is the same.